README.md

```diff
@@ -1,13 +1,237 @@
 # 💪 strong-config
 
-Secret management for NodeJS. Based on [SOPS](https://github.com/mozilla/sops).
+<!-- markdownlint-disable line-length -->
+[![pipeline status](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/pipeline.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master) [![coverage report](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/coverage.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+<!-- markdownlint-enable line-length -->
 
-[![pipeline status](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/pipeline.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+Simple, safe and secure config management for NodeJS. Based on [SOPS](https://github.com/mozilla/sops).
 
-[![coverage report](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/coverage.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+Have you ever struggled with different config values across local, CI,
+and various cluster environments? If no, did you ever forget to update the
+cluster config after updating the local config? Have you ever been worried about
+the secrecy of your DB credentials and API keys taking the `.env` file approach?
+Ever leaked confidential data by pushing secrets in your `.env` file to the git remote?
+
+Do you agree that just `dotenv` is not enough? **`strong-config` is here to help!**
+
+- Structure configs hierarchically by writing JSON or YAML.
+- Manage different configs for different environments, e.g. different clusters
+- Protect your secrets by applying strong cryptography. Encrypt secrets at rest
+  and automatically decrypt them at runtime by integrating a key management
+  service (KMS), e.g. [AWS KMS](https://aws.amazon.com/kms/) or
+  [Google Cloud KMS](https://cloud.google.com/kms/).
+- Enforce arbitrary permission models provided by your KMS. Decide who can encrypt
+  and decrypt configs for which cluster environments.
+- Optionally validate them against a custom schema using
+  [`json-schema`](https://json-schema.org/). Also in CI.
+- Safeguard your configs by using git hooks. Make sure the configs are both
+  valid and encrypted before committing and pushing them.
+- Autogenerate types based on your schema. These types can be imported in your
+  sources to write safer code (Typescript only for now).
+
+These benefits help you to catch inconsistencies, exposed secrets, and invalid
+configs as early as possible in the development process. To do so, you will use
+widely adopted tools you are probably already familiar with: JSON (or YAML),
+your cloud provider, and yet another npm package. Try it out!
+
```

Comment:
So, I elaborated on this part. IMO a bit too much text now early on. When I see a README, I want to have text that is easier to digest. On the other hand, I don't want to make this a science